domains/transfer/Transfer.API/API.IntegrationTests/ClaimAutomation/Api/Controllers/ClaimAutomation20240515ControllerTest.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using System.Net;
+using System.Threading.Tasks;
+using API.IntegrationTests.Factories;
+using FluentAssertions;
+using Xunit;
+
+namespace API.IntegrationTests.ClaimAutomation.Api.Controllers;
+
+[Collection(IntegrationTestCollection.CollectionName)]
+public class ClaimAutomation2030ControllerTest
```

Comment:
I generally prefer a test class to be named after the class it tests, so ex. ClaimAutomation20240515Controller**Test**. I think the file is named like this?

Reply:
ahh yeah. Forgot to update this after I trolled you with versions :D will fix.